src/components/AppBreadcrumb.js

```diff
@@ -36,10 +36,10 @@ const AppBreadcrumb = () => {
   }
 
   const breadcrumbs = getBreadcrumbs(currentLocation)
-
+console.log('networkStatus', networkStatus)
```

Comment:
I'd propose to remove this before merging if possible.

Reply:
ups!